src/main/scala/util/CoreMonitor.scala

```diff
@@ -15,7 +15,7 @@ class CoreMonitorBundle(val xLen: Int) extends Bundle with Clocked {
   val valid = Bool()
   val pc = UInt(width = xLen.W)
   val wrdst = UInt(width = 5.W)
-  val wrdata = UInt(width = xLen.W)
+  val wrdata = UInt(width = 64.W)
```

Comment:
could this be `(xLen max fLen)`?

Reply:
Yes. I'll have to add flen as the parameter. Will update that.